config/_default/menus/menus.en.yaml

```
@@ -354,6 +354,10 @@ main:
     url: "agent/logs/proxy"
     parent: "agent_logs"
     weight: 602
+  - name: "Transport"
+    url: "agent/logs/log-transport"
```

You should update the file name as well, we tend to avoid `-` in file names.

```suggestion
    url: "agent/logs/log_transport"
```